examples/changelog_090.py

```diff
@@ -51,7 +51,13 @@
     'ui_params': {
         'apisSorter': 'alpha',
         'operationsSorter': 'alpha',
```

Comment:
I think you'll want to remove this one now, since it is replaced by the function-version below.